Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml

```
@@ -51,7 +51,7 @@
             - MagazineLightRifleBox
       gun_chamber:
         name: Chamber
-        startingItem: CartridgeLightRifle
+        startingItem: SunriseCartridgeLightRifle #Sunrise-Edit TODO: 5.56
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)